pkg/drivers/kic/oci/volumes.go

```diff
@@ -29,6 +30,47 @@ import (
 	"k8s.io/klog/v2"
 )
 
+// RemoveVolume removes a volume
+func RemoveVolume(ociBin string, name string) error {
+	if !volumeExists(ociBin, name) {
```

Comment:
nit: what is the performance implication of running this ? would it be better if we just try remove it regardless if it exists or not ? and if it errors out we could check if error is Volume doesn't exist?

Reply:
Most of the code is just copy/pasted from the network, there are no specific performance considerations. The volume is normally removed with the container (the `-v`) so it is just fallback really.